File: Data Structures & Algorithms/two-integer-sum-ii/submission-1.swift
class Solution {
    func twoSum(_ arr: [Int], _ target: Int) -> [Int] {
        var left = 0
        var right = arr.count - 1
        while(left < right){
            let sum = arr[left] + arr[right]
            if sum == target {
                return [left + 1, right + 1]
            } else if (sum < target){
                left += 1
            } else {
                right -= 1
            }
        }
        return[]
    }
}

File: Data Structures & Algorithms/two-integer-sum/submission-1.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        map: Dict = {}
        for i in range(len(nums)):
            num = target - nums[i]
            if num in map:
                return [map[num],i]
            map[nums[i]] = i
        return []
        
        

        